.github/workflows/build-lint-test.yml

```
@@ -16,9 +16,48 @@ jobs:
       - run: git diff --quiet || { echo 'working directory dirty after "yarn build"'; exit 1; }
       - run: yarn lint
       - run: yarn test
+  # tests to ensure get-updated-packages.sh functions as expected
+  updated-packages-test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Get Latest Version from npm
+        id: latestrelease
+        run: echo "::set-output name=releasever::$(npm view MetaMask/snaps-skunkworks version --workspaces=false)"
+      - uses: actions/checkout@v3
+        with:
+          repository: MetaMask/snaps-skunkworks
+          ref: v${{ steps.latestrelease.outputs.releasever }}
+          path: skunkworks
+      - uses: actions/checkout@v3
+        with:
+          path: action-publish-release
+      - name: Modify + Get UPDATED_PACKAGES lengths
+        id: modify-get-updated-packages
+        run: |
+          cd skunkworks
```

Shellcheck suggests `|| exit` here in case `cd` fails

```suggestion
          cd skunkworks || exit
```